src/aot/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-add_library(aot aot.cpp)
+add_library(aot aot.cpp ${CMAKE_SOURCE_DIR}/src/required_resources.cpp)
```

Comment:
Yeah this is a bit unfortunate. I feel like cmakelists.txt should try not to reach up to parent  or sibling dirs

Reply:
I don't like it either. But I couldn't come with any other solution, besides adding the dummy constructor (see the other thread).

Reply:
Should required_resources.cpp live in the aot directory, since that's where it's used?

Reply:
Possibly but we wouldn't have `required_resources.cpp` next to `required_resources.h`.

Reply:
I was thinking of moving the header too, but I suppose it's used in more than just AOT so that doesn't make sense.

How about creating a new CMake library target just for required_resources? Then both runtime and aot can link against that. Would it solve the unused symbol issues?

Reply:
Agreed - separate target is probably cleanest solution

Reply:
Good idea, done.